napari/_app_model/constants/_commands.py

```diff
@@ -95,6 +102,11 @@ class _i(NamedTuple):
     CommandId.TOGGLE_VIEWER_SCALE_BAR_COLORED: _i(trans._('Scale Bar Colored')),
     CommandId.TOGGLE_VIEWER_SCALE_BAR_TICKS: _i(trans._('Scale Bar Ticks')),
 
+    # Help menubar
+    CommandId.NAPARI_INFO: _i(trans._('napari Info'), ),
+    CommandId.TOGGLE_BUG_REPORT_OPT_IN: _i(trans._('Bug reporting opt in/out...'), ),
```

Comment:
I think we decided titlecase for all menu items.

```suggestion
    CommandId.TOGGLE_BUG_REPORT_OPT_IN: _i(trans._('Bug Reporting Opt In/Out...'), ),
```

Reply:
I probably copied this from the old class but happy to change it for consistency